Model/inverted-index.js

```diff
@@ -113,28 +103,44 @@ class Index {
     }
   }
 
-  searchIndex(terms, fileName) {
-    const selectedIndex = this.getIndex(fileName);
+  searchIndex(sTerms, fileName) {
```

Comment:
Use descriptive names for variables and parameters, I'm able to understand that `sTerms` means `searchTerms` cos I'm quite familiar with the program but what happens to a strange developer reading your code?.